### PR TITLE
Prep for 8.6 Branch Creation

### PR DESCRIPTION
**Pull request by terrancedejesus**
## Related
* https://github.com/elastic/ia-trade-team/issues/73

## Summary
- [x] update packages.yml
- [x] update integrations manifest
- [x] update lock-versions.yml
- [x] update workflows
- [x] lock api_schemas
- [x] update beats and ecs schemas
- [x] add migrate function
- [x] update stack-schema-map

_merged after 8.5 branch creation_

### CODE REVIEW

**Comment by terrancedejesus** [comment]
Note -> The backporting branch check is expected to fail for `8.5` because the branch is created right before the merge.

File: detection_rules/etc/stack-schema-map.yaml [hunk 1]
```diff
@@ -51,3 +51,7 @@
 "8.5.0":
   beats: "main"
   ecs: "8.3.1"
+
+"8.6.0":
+  beats: "main"
+  ecs: "8.4.0"
```
Comment (Mikaayenson) [inline detection_rules/etc/stack-schema-map.yaml:57]: Should this be `8.5.0`?

Comment (brokensound77) [reply detection_rules/etc/stack-schema-map.yaml:57]: 8.3 as well

File: detection_rules/schemas/__init__.py [hunk 2]
```diff
@@ -212,6 +212,12 @@ def migrate_to_8_4(version: Version, api_contents: dict) -> dict:
     return strip_additional_properties(version, api_contents)
 
 
+@migrate("8.5")
```
Comment (brokensound77) [inline detection_rules/schemas/__init__.py:215]: May need to verify the invalidation of _all_ restricted fields and rule types for downgrades. Maybe punt that to a separate issue

File: detection_rules/etc/stack-schema-map.yaml (hunk 1 above)
Comment (terrancedejesus) [reply detection_rules/etc/stack-schema-map.yaml:57]: We can update it later, but at the moment, `8.4.0` is the latest, while `8.5.0` is a release candidate as of today.

File: detection_rules/schemas/__init__.py (hunk 2 above)
Comment (terrancedejesus) [reply detection_rules/schemas/__init__.py:215]: Noted and agreed. Added an issue for this that can be solved in a separate PR.

* https://github.com/elastic/detection-rules/issues/2320

**Review by Mikaayenson — APPROVED** [review]
LGTM

**Review by imays11 — APPROVED** [review]
👍🏾

**Pull request merged**